BREEZE.rst

```diff
@@ -79,11 +79,9 @@ images maintained on the Docker Hub in the ``apache/airflow`` repository.
 
 There are three images that we are currently managing:
 
-* **Slim CI** image that is used for static code checks (size of ~500MB). Its tag follows the pattern
-  of ``<BRANCH>-python<PYTHON_VERSION>-ci-slim`` (for example, ``apache/airflow:master-python3.6-ci-slim``).
-  The image is built using the `<Dockerfile>`_ Dockerfile.
-* **Full CI image*** that is used for testing. It contains a lot more test-related installed software
-  (size of ~1GB). Its tag follows the pattern of ``<BRANCH>-python<PYTHON_VERSION>-ci``
+* **Full CI image*** that is used for testing od both Unit tests and static check tests.
```

Comment:
Maybe we should call this image "CI Image" (without full) now ?

Reply:
Yep.